mne/channels/channels.py

```diff
@@ -559,8 +559,9 @@ def drop_channels(self, ch_names):
 
         Parameters
         ----------
-        ch_names : list
-            The list of channels to remove.
+        ch_names : str | list
+            Name of the channel to remove, or the list of channels to
+            remove.
```

Comment:
Yes, but forgetting it is very common, isn't it? We also do this with picks very often.